.copier-answers.yml

```diff
@@ -0,0 +1,22 @@
+# Autogenerated. Do not edit this by hand, use `copier update`.
+---
+_commit: 0.2.6
+_src_path: https://github.com/lkubb/salt-extension-copier
+author: EITR Technologies, LLC
+author_email: eitr@devops.tech
+docs_url: ''
+license: apache
+loaders:
+  - module
+  - state
```

Comment:
Just a note: It's always a good idea to ensure the answers fit reality, otherwise one might be missing out on some automation. This should contain `cache`, `pillar` and `sdb` as well I think.